crates/mfem-sys/src/ffi_autocxx.hpp

```diff
@@ -44,6 +44,10 @@ namespace acxx {
     SUBCLASS(SparseSmoother, MatrixInverse)
     SUBCLASS(GSSmoother, SparseSmoother)
 
+    void Vector_set_all(Vector& vector, real_t value) {
+        vector = value;
+    }
+
```

Comment:
Not needed if we deref vectors into slices `[f64]`.

Reply:
Good point, then we can simply use [.fill()](https://doc.rust-lang.org/std/primitive.slice.html#method.fill).